scripts/jest/shouldIgnoreConsoleError.js

```diff
@@ -23,6 +27,15 @@ module.exports = function shouldIgnoreConsoleError(format, args) {
         // We haven't finished migrating our tests to use createRoot.
         return true;
       }
+      if (
+        TODO_ignoreHydrationErrors &&
+        format.indexOf(
+          'An error occurred during hydration. The server HTML was replaced with client content in'
+        ) !== -1
+      ) {
+        // This also gets logged by onRecoverableError, so we can ignore it.
+        return true;
```

Comment:
There is still one error that is duplicated in console.error and `onRecoverableError`, so it's fine to ignore. There's a comment on that error to remove it: https://github.com/facebook/react/blob/763612647ceb66d95f728af896ca5e18a8181db8/packages/react-dom-bindings/src/client/ReactFiberConfigDOM.js#L1682